test/lib-publish.spec.js

```diff
@@ -60,6 +63,35 @@ describe('skyux-deploy lib publish', () => {
     );
 
     expect(publishSpaMock).not.toHaveBeenCalled();
+    expect(loggerMock.info).toHaveBeenCalledWith(loggerInfoSuccess);
+  });
+
+  it('should log an error when settings.isStaticClient = false', async () => {
+    const error = 'some-error-happened-is-static-client-false';
+    publishSpaMock.and.callFake(() => {
+      throw error;
+    });
+
+    try {
```

Comment:
I think you want to call `await expectAsync(...).toBeRejectedWith(...)` here.  The problem with the test as it's written is that if `callPublish()` doesn't throw an error, then no `expect`s get called and the test will pass.  See `lib-deploy.spec.js` for examples of how to use it. https://github.com/blackbaud/skyux-deploy/blob/master/test/lib-deploy.spec.js#L73

Reply:
Yup, I didn't a bad job of searching for `toBeRejectedWith`.  I'll update.